kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/TableConfigSuite.scala

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
I feel like this is not needed as the same code will get tested as part of the E2e tests. Let me know.